packages/core/src/Serialization/CBOR/CborWriter.ts

```diff
@@ -28,6 +28,15 @@ const BUF_INF_POS = Buffer.from('7ff0000000000000', 'hex');
 
 /** A simple writer for Concise Binary Object Representation (CBOR) encoded data. */
 export class CborWriter {
+  /**
+   * Enable encoding `set` type as 6.258, according to the Conway era CDDL.
+   * https://github.com/IntersectMBO/cardano-ledger/blob/master/eras/conway/impl/cddl-files/extra.cddl#L5
+   *
+   * Enable encoding `redeemers` as `Map`:
+   * https://github.com/IntersectMBO/cardano-ledger/blob/master/eras/conway/impl/cddl-files/conway.cddl#L480
+   */
+  static useConwaySerialization = false;
```

Comment:
another nitpick, This "feature flag" seems a bit out of place here, the CborWriter is general purpose and should be agnostic to Cardano domain concepts. Also the writer doesn't really do anything different based on this flag, maybe would be better to hoist this to a different module?. But again, just a small nitpick since this is temporary